hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/scm/TestGetCommittedBlockLengthAndPutKey.java

```diff
@@ -110,12 +115,21 @@ public void tesGetCommittedBlockLength() throws Exception {
         ContainerTestHelper
             .getPutBlockRequest(pipeline, writeChunkRequest.getWriteChunk());
     client.sendCommand(putKeyRequest);
-    response = ContainerProtocolCalls
-        .getCommittedBlockLength(client, blockID, null);
+    GenericTestUtils.waitFor(() -> {
+      boolean success = true;
+      try {
+        response[0] = ContainerProtocolCalls
+            .getCommittedBlockLength(client, blockID, null);
+      } catch (IOException e) {
+        success = false;
+        LOG.error("Ignore the exception till wait: {}", e.getMessage());
+      }
+      return success;
+    }, 1000, 5000);
     // make sure the block ids in the request and response are same.
     Assertions.assertTrue(
-        BlockID.getFromProtobuf(response.getBlockID()).equals(blockID));
-    Assertions.assertTrue(response.getBlockLength() == data.length);
+        BlockID.getFromProtobuf(response[0].getBlockID()).equals(blockID));
+    Assertions.assertTrue(response[0].getBlockLength() == data.length);
```

Comment:
This should be ``assertEquals`` I believe rather than doing equals inside ``assertTrue``

```
    Assertions.assertEquals(BlockID.getFromProtobuf(response[0].getBlockID()), blockID);
    Assertions.assertEquals(response[0].getBlockLength(), data.length);
```

Reply:
Thanks @ayushtkn for review, I have changed it to assertEquals.